fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/ReindexOperationTest.java

```diff
@@ -14,27 +14,41 @@
 import java.time.ZonedDateTime;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Properties;
 import java.util.UUID;
 
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
 
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import com.ibm.fhir.core.FHIRMediaType;
 import com.ibm.fhir.model.resource.Parameters;
 import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.test.TestUtil;
 import com.ibm.fhir.server.test.FHIRServerTestBase;
 
 /**
  * This integration tests the <b>fhir-operation-reindex</b> $reindex operation.
  * These reindex test always run.
```

Comment:
we probably want to remove this line before merging :-)

Reply:
just changed it back to run, and left the code to disable in place, just in case we revert back